mleap-executor-testkit/src/main/scala/ml/combust/mleap/executor/testkit/TransformServiceSpec.scala

```diff
@@ -361,7 +361,8 @@ trait TransformServiceSpec extends FunSpecLike
       }
 
       describe("throttling") {
-        it("throttles elements through the flow") {
+        // TODO this test fails randomly on Travis, so temporarily replaced it() with ignore()
```

Comment:
Do you have a link to an example of a build failure ? I can take a quick look

Reply:
@lucagiovagnoli 

https://github.com/combust/mleap/pull/568#issuecomment-532371321
https://github.com/combust/mleap/pull/568#discussion_r327019642

This commit was actually merged as part of https://github.com/combust/mleap/pull/568 so will disappear from this diff after rebasing.

Should continue discussion about the flake at https://github.com/combust/mleap/pull/568